views/templates/block/channelInfo.twig

```diff
@@ -1,47 +1,82 @@
+                <div class="col-xs-3 col-md-1">
+                    <label class="label-small">{{ 'Version number:'|trans }}</label>
+                </div>
+                <div class="col-xs-9 col-md-11">  
+                    <span class="version">{{ upgradeInfo.version_num }}</span>
+                </div>
             </div>
         {% endif %}
         {% if upgradeInfo.link is not empty %}
-            <div style="clear:both;">
-                <label class="label-small">{{ 'URL:'|trans }}</label>
-                <a class="url" href="{{ upgradeInfo.link }}">{{ upgradeInfo.link }}</a>
+            <div class="row">
+                <div class="col-xs-3 col-md-1">
+                    <label class="label-small">{{ 'URL:'|trans }}</label>
+                </div>
+                <div class="col-xs-9 col-md-11">
+                    <a class="url" href="{{ upgradeInfo.link }}">{{ upgradeInfo.link }}</a>
+                </div>    
             </div>
         {% endif %}
         {% if upgradeInfo.md5 is not empty %}
-            <div style="clear:both;">
-                <label class="label-small">{{ 'MD5 hash:'|trans }}</label>
-                <span class="md5">{{ upgradeInfo.md5 }}</span>
+            <div class="row">
+                <div class="col-xs-3 col-md-1">
+                    <label class="label-small">{{ 'MD5 hash:'|trans }}</label>
+                </div>
+                <div class="col-xs-9 col-md-11">  
+                    <span class="md5">{{ upgradeInfo.md5 }}</span>
+                </div>
             </div>
         {% endif %}
         {% if upgradeInfo.changelog is not empty %}
-            <div style="clear:both;">
-                <label class="label-small">{{ 'Changelog:'|trans }}</label>
-                <a class="changelog" href="{{ upgradeInfo.changelog }}">{{ 'see changelog'|trans }}</a>
+            <div class="row">
+                <div class="col-xs-3 col-md-1">
+                    <label class="label-small">{{ 'Changelog:'|trans }}</label>
+                </div>
+                <div class="col-xs-9 col-md-11">  
+                    <a class="changelog" href="{{ upgradeInfo.changelog }}">{{ 'see changelog'|trans }}</a>
+                </div>
             </div>
         {% endif %}
+        <div class="row">
+            <div class="col-xs-3 col-md-1">
+                <label class="label-small">{{ 'PHP compatibility:'|trans }}</label>
+            </div>
+            <div class="col-xs-9 col-md-11">
+                <span class="phpcompatibility">{{ 'Your PHP version must be compatible with both your current PrestaShop version and the one you want to upgrade to.
+                    For example, if your current version is 1.6.1.x and you want to upgrade to 1.7.8, only the PHP 7.1 would be compatible with both.'|trans }}
+                </span>
+                <img class="img-responsive" src="../modules/autoupgrade/views/img/php-comptibility-table.png" width="1000">
```

Comment:
We might use an image a little bit more pretty 👀  this one is a screenshot, right?

It would be even better to output a HTML table. Images are not good for accessibility 😉  and HTML handle very well tables.

Alternatively we could simply redirect to https://devdocs.prestashop.com/1.7/basics/installation/system-requirements/#system-requirements-for-prestashop-17 . So if in the future we update the PHP version table on devdocs, we won't have to release a new version of this module just to update the table.

Reply:
I follow https://www.figma.com/file/GpRsUUCHcdhJPAaz7Eqzes/Autoupgrade?node-id=497%3A11710 but i agree

Reply:
I'm quite sure this image is supposed to be a dynamic table rendered by PHP or JS. The lines highlighted on the image are 1.6.1.x and 1.7.8 and I think these are dynamic elements that adapt to the situation.

Reply:
@marionf say : 

_FYI, the table is static and given as an example: it does not change depending on the version of php or on the targeted version of PS selected for the upgrade._

Reply:
Then I will have a discussion with @marionf but I will not approve a PR which uses an image where 1.6.1.x is marked as 'current' and 1.7.8 is marked as 'wanted'.

Let's say I approve this PR and we release this module. So if tomorrow someone wants to upgrade from 1.7.1 to 1.7.7, he'll see an image where 1.6.1.x is marked as 'current' and 1.7.8 is marked as 'wanted' . This is highly confusing. I think there is a misunderstanding here 😄  . My guess is that this image is not the right one. Maybe if we remove the 'current' and 'wanted' words from the image it would work.

Additionally this image cannot be translated so if a German user or a Greek user opens the module, he will see a module with German and Greek explanations but an image in English.

Something is wrong here 😊  let's find what. I'm quite sure there is a misunderstanding.

Reply:
Hello @matks i do this it's now translatable.

I think the current and wanted is to make it clear to the merchant.

I hate table :angry: lol

![Capture d’écran de 2021-09-24 09-17-49](https://user-images.githubusercontent.com/5064590/134634600-1c71b56e-1fa3-4831-800c-25020f53337a.png)

Reply:
It's better to add the next PrestaShop versions

Reply:
That's an example and it's indicated just before the table: "For example, if your current version is 1.6.1.x and you want to upgrade to 1.7.8, only the PHP 7.1 would be compatible with both."

Reply:
> Additionally this image cannot be translated so if a German user or a Greek user opens the module, he will see a module with German and Greek explanations but an image in English.

We should not use an image but well a table :)